src/client/cli/commands/publishRaw.js

```diff
@@ -4,9 +4,9 @@ const RestClient = require('../../api/RestClient')
 
 module.exports = {
   command: 'publishRaw <namespace> <statementBodyId>',
-  description: 'publish a statement whose body (actual metadata content) has \n' +
-    'already been stored in the node.  `statementBodyId` should be the multihash \n' +
-    'identifier of the statement body,',
+  description: 'publish a statement whose body (actual metadata content) has ' +
```

Comment:
I think the metadata/statement distinction is something we really need to make clear